test/unittest/x86-64-linux/WasiTest.v3

```diff
@@ -0,0 +1,44 @@
+// Copyright 2023 Wizard authors. All rights reserved.
+// See LICENSE for details of Apache 2.0 license.
+
+
+def T = UnitTests.registerT("wasi:", _, WasiTester.new, _);
+def X = [
+	T("normalize_path", test_normalize_path),
```

Comment:
It's also useful sometimes to split this into multiple different unit tests with only a couple cases each. That speeds up debugging, should one fail in the future, by narrowing down what's failed.